src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let callCount = 0;
+  const sum = (a, b) => a + b;
+  const error = () => 'Bzzz... Error!';
+
+  function getSum(a) {
+    callCount++;
+
+    if (callCount > 3 && callCount % 2 === 0) {
+      return error;
```

Comment:
I think you should call error here or change error to plain string
```suggestion
      return error()
```